### ISSUE TITLE
There seems to be something wrong with tree shaking 

### ISSUE THREAD

**Issue by germsb** [vueuse/vueuse#469]
When I `import {useFocusTrap} from "@vueuse/integrations"`, all the lib is bundled  (qrcode, buffer & co .. ).
[repro](https://github.com/germsb/vueuse-tree-shake)  `npm install`, `npm run dev` See stats.html in dist after build.

**Comment by antfu** [comment]
Oh, that's a good point and I didn't notice that as I thought rollup will tree-shake unused modules. But looks like there is also no way to explicitly mark an import statement pure (https://github.com/rollup/rollup/issues/2874). The solution of this that comes up my mind is to separate functions into several sub-modules, e.g.

```ts
import {useFocusTrap} from "@vueuse/integrations/useFocusTrap"
import {useQRCode} from "@vueuse/integrations/useQRCode"
```

What do you think?

If we go that way, it will be a breaking change and I would like to do that in v5.

**Comment by germsb** [comment]
Hello, it is true that it is quite difficult to manage.
In short, yes, it's probably the easiest way and the the only way for cdn.

If you want to keep the actual syntax for bundler, share the src entry point where you export all the utilities 
```js
//index.ts
export {useFocusTrap} from "./src/integrations/useFocusTrap.ts"
export {useQRCode} from "./src/integrations/useQRCode.ts"
```
###### it's just a deduction, not sure it's the right way

**Comment by wheatjs** [comment]
Has this been addressed in 5.0 yet @antfu?

**Comment by antfu** [comment]
emm, not yet, but good point. want to give it a try? 👀

**Comment by wheatjs** [comment]
Sure, I'll try to get a PR going in a bit

**Comment by antfu** [comment]
@wheatjs I will do this now

**Comment by antfu** [comment]
Available in `5.0.0-beta.7`

```ts
import { useFocusTrap } from "@vueuse/integrations/useFocusTrap"
import { useQRCode } from "@vueuse/integrations/useQRCode"
```

The existing usage is still supported and remains the same behavior and tree-shaking limitations  

```ts
// work, but no longer recommended
import { useFocusTrap, useQRCode } from "@vueuse/integrations"
```

**Comment by milesingrams** [comment]
When using the  `import { UseFocusTrap } from '@vueuse/integrations/useFocusTrap/component'  in a Nuxt 3 project with `@vueuse/integrations: 7.3.0` I'm getting the error:

```
SyntaxError: The requested module '/_nuxt/node_modules/qrcode/lib/browser.js?v=93c8fa7e' does not provide an export named 'default'
```

It appears that two things may be going on:
1. Tree shaking is not happening and therefore the `useQRCode` composable is being loaded
2. Somehow the `useQRCode` composable is possibly incorrectly importing `qrcode`

However this also may be an issue in Nuxt 3 and how it performs imports?
Any help would be much appreciated!
